tests/test_connection.py

```diff
@@ -5,41 +5,36 @@
 import datajoint as dj
 from datajoint import DataJointError
 import numpy as np
-from . import CONN_INFO_ROOT
-from . import PREFIX
 import pytest
 
 
+class Subjects(dj.Manual):
+    definition = """
+    #Basic subject
+    subject_id                  : int      # unique subject id
+    ---
+    real_id                     :  varchar(40)    #  real-world name
+    species = "mouse"           : enum('mouse', 'monkey', 'human')   # species
+    """
+
+
 @pytest.fixture
-def schema(connection_test):
+def schema_tx(connection_test, prefix):
     schema = dj.Schema(
-        PREFIX + "_transactions", context=dict(), connection=connection_test
+        prefix + "_transactions",
+        context=dict(Subjects=Subjects),
+        connection=connection_test,
     )
+    schema(Subjects)
     yield schema
     schema.drop()
 
 
-@pytest.fixture
-def Subjects(schema):
-    @schema
-    class Subjects(dj.Manual):
-        definition = """
-        #Basic subject
-        subject_id                  : int      # unique subject id
-        ---
-        real_id                     :  varchar(40)    #  real-world name
-        species = "mouse"           : enum('mouse', 'monkey', 'human')   # species
-        """
-
-    yield Subjects
-    Subjects.drop()
-
-
```

Comment:
Moved `Schema` as a module level definition. This keeps it consistent with other schemas in the test suite.